app/models/concerns/multiparameter_attribute_assignment.rb

```diff
@@ -0,0 +1,202 @@
+# https://github.com/rails/rails/pull/8189#issuecomment-32803107
+
+module MultiparameterAttributeAssignment
+  include ActiveModel::ForbiddenAttributesProtection
+
+  def initialize(params = {})
+    assign_attributes(params)
+  end
+
+  def assign_attributes(new_attributes)
+    multi_parameter_attributes = []
+
+    attributes = sanitize_for_mass_assignment(new_attributes.stringify_keys)
+
+    attributes.each do |k, v|
```

Comment:
Style: single-letter iterator values. Maybe `key` and `value`? or `attribute` and `value`? Not a huge deal.